tests/test_kernels.py

```diff
@@ -76,105 +72,75 @@ def test_required_grid_vars(grid_type_field_and_extra_kwargs):
 
 # The following definition of irregular_grids is hard coded; maybe a better definition
 # would be: all grids that have len(required_grid_vars)>1 (more than just a wet_mask)
-irregular_grids = [
-    gt for gt in GridType if "IRREGULAR" in gt.name or "TRIPOLAR_POP" in gt.name
-]
+irregular_grids = [GridType.IRREGULAR_WITH_LAND, GridType.TRIPOLAR_POP_WITH_LAND]
 
 
-def test_flux_in_y_direction(grid_type_field_and_extra_kwargs):
+@pytest.mark.parametrize("direction", ["X", "Y"])
+def test_flux_in_direction(grid_type_field_and_extra_kwargs, direction):
```

Comment:
Minor detail: Now that the flux tests for the x- and y-directions are combined, we should change the docstring of this test function and remove anything specific to the y-direction test. We could also remove the `_in_direction` from the name of the test function.